tests/acceptance/characters/test_transacting_power.py

```diff
@@ -41,7 +41,8 @@ def test_character_transacting_power_signing(testerchain, test_registry, account
 
     # Sign Message
     data_to_sign = b"Premium Select Luxury Pencil Holder"
-    signature = power.sign_message(message=data_to_sign)
+    _message_hash, signature = power.sign_message(message=data_to_sign)
```

Comment:
I assume `_message_hash` is unused. Did you want to do?:

```suggestion
    _, signature = power.sign_message(message=data_to_sign)
```
(other places too)

Reply:
My bad - I guess this is PEP 640.